x-pack/plugins/task_manager/server/routes/metrics.ts

```diff
@@ -63,6 +66,11 @@ export function metricsRoute(params: MetricsRouteParams) {
       req: KibanaRequest<unknown, TypeOf<typeof QuerySchema>, unknown>,
       res: KibanaResponseFactory
     ): Promise<IKibanaResponse> {
+      debugLogger.debug(
```

Comment:
I wonder if we should also debug log in the `metrics$.subscribe((metrics) = > ...` function in case there's a gap between setting the source of truth (`task_run_metrics_aggregator`) and the `lastMetrics` object being updated.

Reply:
That will log the metrics block every 3 seconds when debug logging is turned on...do you think that's too noisy?

Reply:
I think it's ok if we plan to only have debug logs enabled for a few hours. Otherwise, maybe not.

Reply:
i will put it under a different context so we can enable them separately

Reply:
Updated in https://github.com/elastic/kibana/pull/184159/commits/a743b2cc6b94e1f96a58f2b27d96dd518ab3e1b8.